app/controllers/prx_auth/rails/sessions_controller.rb

```diff
@@ -55,10 +50,29 @@ def create
         sign_in_user(access_token)
         redirect_to after_sign_in_path_for(current_user)
       else
+        session.delete(WILDCARD_SESSION_KEY)
         redirect_to auth_error_sessions_path(error: params[:error] || "unknown_error")
       end
     end
 
+    def destroy
+      sign_out_user
+      redirect_to after_sign_out_path, allow_other_host: true
+    end
+
+    def logout
+      sign_out_user
+      redirect_to "//#{id_host}/session/sign_out", allow_other_host: true
```

Comment:
I'm a little confused what is different in destroy and logout?

Reply:
Hmm, yeah, it appears `logout` was added later to our apps.  Because (A) it's a `GET` instead of a `DELETE`, so you don't need a button_to or anything.  And (B) it also logs you out of ID.